templates/lxc-download.in

```diff
@@ -83,26 +83,26 @@ wget_wrapper() {
 }
 
 download_file() {
-    if ! wget_wrapper -T 30 -q https://${DOWNLOAD_SERVER}/$1 -O $2 >/dev/null 2>&1; then
-        if ! wget_wrapper -T 30 -q http://${DOWNLOAD_SERVER}/$1 -O $2 >/dev/null 2>&1; then
+    if ! wget_wrapper -T 30 -q "https://${DOWNLOAD_SERVER}/$1" -O "$2" >/dev/null 2>&1; then
+        if ! wget_wrapper -T 30 -q "http://${DOWNLOAD_SERVER}/$1" -O "$2" >/dev/null 2>&1; then
             if [ "$3" = "noexit" ]; then
                 return 1
             else
                 echo "ERROR: Failed to download http://${DOWNLOAD_SERVER}/$1" 1>&2
                 exit 1
             fi
-        elif [ "$DOWNLOAD_SHOW_HTTP_WARNING" = "true" ]; then
+        elif [ "${DOWNLOAD_SHOW_HTTP_WARNING}" = "true" ]; then
             DOWNLOAD_SHOW_HTTP_WARNING="false"
             echo "WARNING: Failed to download the file over HTTPs." 1>&2
-            echo -n "         The file was instead download over HTTP. " 1>&2
```

Comment:
Because according to shellcheck
```
In lxc-download.in line 103:
            echo -n "         The file was instead download over HTTP. " 1>&2
                 ^-- SC2039: In POSIX sh, echo flags are undefined.
```